### PR TITLE
update env var for logging testing

### PR DESCRIPTION
**Pull request by QiaolingTang**
https://issues.redhat.com/browse/OCPQE-3595 
Adding env  var `logging_sub_envs`
```
logging_sub_envs:
  clo:
    catsrc: "test"
    channel: "4.1"
  eo:
    catsrc: "redhat"
    channel: "5.1"`
```

/cc @anpingli

### CODE REVIEW

File: features/step_definitions/logging.rb [hunk 1]
```diff
@@ -326,36 +327,75 @@
   })
 end
 
-Given /^logging channel name is stored in the#{OPT_SYM} clipboard$/ do | cb_name |
-  cb_name = 'logging_channel_name' unless cb_name
-  if env.logging_channel_name.empty?
+# start from logging 5.0, we use our own CI pipeline to trigger automation testing
+# add env vars to set channel and catsrc separately for CLO and EO
+# for example:
+# logging_sub_envs:
```
Comment (anpingli) [inline features/step_definitions/logging.rb:333]: @QiaolingTang  We may pass more parameters(For example: eventrouter image name) into cucushift in future. Could we use  more common name logging_env?

Comment (QiaolingTang) [reply features/step_definitions/logging.rb:333]: OK

**Review by anpingli — COMMENTED** [review]
/lgtm

**Pull request merged**